packages/shared/src/components/sidebar/SidebarBottomSection.tsx

```diff
@@ -8,6 +10,11 @@ import InvitePeople from './InvitePeople';
 import { Section, SectionCommonProps } from './Section';
 import { docs, feedback } from '../../lib/constants';
 
+const ChangelogModal = dynamic(
```

Comment:
Can you please use LazyModal instead ?

Reply:
Moved away from modal on this component but gonna keep in mind for other.